### PR TITLE
DDF-2335 Disable the Hazelcast version check on startup.

### PR DESCRIPTION
**Pull request by coyotesqrl**
#### What does this PR do?
Disables the remote version check

#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@ryeats, @stustison 

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@coyotesqrl
@pklinef

#### How should this be tested?
Build and unzip; run a full install of the app and see that the erroneous `fatal error` message no longer appears.

#### Any background context you want to provide?
N/A

#### What are the relevant tickets?
DDF-2335

#### Screenshots (if appropriate)
N/A

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by pklinef** [comment]
👍

File: distribution/ddf-common/src/main/resources/etc/system.properties [hunk 1]
```diff
@@ -173,3 +173,8 @@ karaf.secured.services=(&(osgi.command.scope=*)(osgi.command.function=*))
 #org.osgi.framework.trust.repositories=${karaf.home}/etc/trustStore.ks
 ws-security.subject.cert.constraints = .*
 security.audit.roles=group,admin,manager,viewer,system-admin,system-history,systembundles
+
+#
+# Disable Hazelcast version check
+#
+hazelcast.version.check.enabled=false
```
Comment (pklinef) [inline distribution/ddf-common/src/main/resources/etc/system.properties]: Looks like this property might be deprecated.  Not sure if it was in the version of Hazelcast we are using and what the right way is to disable version checking for newer versions.

https://github.com/hazelcast/hazelcast/blob/167361456fe0b8d77b57aae72dd415f8ade56f61/hazelcast/src/main/java/com/hazelcast/spi/properties/GroupProperty.java#L157-L159

Comment (pklinef) [inline distribution/ddf-common/src/main/resources/etc/system.properties]: Looks like the documentation still includes that option.  Maybe just that constant in that class has been deprecated.

http://docs.hazelcast.org/docs/3.5/manual/html/systemproperties.html

**Comment by pklinef** [comment]
👍

**Comment by coyotesqrl** [comment]
Rebuilt locally to confirm the Bamboo failure was unrelated and not reproducible.

**Pull request merged**